src/yea/runner.py

```diff
@@ -350,6 +349,17 @@ def finish(self) -> None:
             print("  {:<{}s}: {}{}".format(tc.name, tlen, emoji, r))
             if r:
                 exit_code = 1
+
+        # timing info
+        print("\nTest durations (sec):")
+        print("---------------------")
+        timing_info = sorted(
+            ((tc.elapsed_sec, tc.name) for tc in self._results),
+            reverse=True,
+        )
+        for tc in timing_info:
+            print(f"  {tc[1]:<{tlen}s}: {tc[0]:.1f}")
```

Comment:
shouldn't it be the other way around. I thought the tuple is `(tc.elapsed_sec, tc.name)` i.e. first elapsed time and second run name?
```suggestion
            print(f"  {tc[0]:<{tlen}s}: {tc[1]:.1f}")
```

Reply:
yea, that's right, it's `(tc.elapsed_sec, tc.name)` (for sorting), but I here I print the name first and the duration on the right.